types/react-autosuggest/index.d.ts

```diff
@@ -5,84 +5,200 @@
 //                 Robert Essig <https://github.com/robessog>
 //                 Terry Bayne <https://github.com/tbayne>
 //                 Christopher Deutsch <https://github.com/cdeutsch>
+//                 Kevin Ross <https://github.com/rosskevin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 import * as React from 'react';
 
-declare class Autosuggest extends React.Component<Autosuggest.AutosuggestProps> {}
+declare class Autosuggest<T> extends React.Component<Autosuggest.AutosuggestProps<T>> {}
```

Comment:
Would changing this to `declare class Autosuggest<T = any>` make this backward compatible?

Reply:
Added, tests work.